app/scripts/controllers/metametrics-controller.ts

```diff
@@ -782,12 +925,10 @@ export default class MetaMetricsController {
    * @param payload - details of the event
    * @param options - options for handling/routing the event
    */
-  async submitEvent(
+  async #submitEvent(
     payload: MetaMetricsEventPayload,
     options?: MetaMetricsEventOptions,
   ): Promise<void> {
-    this.validatePayload(payload);
```

Comment:
It seems that this line was deleted, do we still need this?

Reply:
No, we don’t need this because submitEvent is already called within trackEvent before this method which does validate the payload. I believe it was initially added as a workaround to test submitEvent, which wasn’t marked as private, but I’ve now corrected that. I’ve moved all the tests to the parent public method.